CMakeCommon.cmake

```diff
@@ -104,8 +104,11 @@ function (LinkGSLibrariesToProject acVersion devKitDir addOnName)
         file (GLOB LibFilesInFolder ${devKitDir}/Modules/*/*/*.lib)
         target_link_libraries (${addOnName} ${LibFilesInFolder})
     else ()
-        file (GLOB FrameworkFilesInFolder ${devKitDir}/Frameworks/*.framework)
-        target_link_libraries (${addOnName} ${FrameworkFilesInFolder})
+        file (GLOB LibFilesInFolder
+            ${devKitDir}/Frameworks/*.framework
```

Comment:
Ákos Somorjai just removed the frameworks from the DevKit because they are unnecessary for building the add-on. It also decreased the mac package size by 400Mbs We should not add them again, dylibs should be enough.